src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -207,11 +208,12 @@ class SaveInProgressIntro extends React.Component {
   };
 
   goToBeginning = () => {
+    recordEvent({ event: 'no-login-start-form' });
```

Comment:
This handles the `CTA 2` case defined here: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16807.